src/react-native/hoc.js

```diff
@@ -7,9 +7,28 @@ import { Events } from '../shared/constants';
 export const withMessaging = (WebView) => {
   return class WebViewWithMessaging extends React.PureComponent {
     render() {
+      const patchPostMessageFunction = () => {
+        var originalPostMessage = window.postMessage;
+        var patchedPostMessage = function(message, targetOrigin, transfer) {
+          originalPostMessage(message, targetOrigin, transfer);
+        };
+
+        patchedPostMessage.toString = () => {
+          return String(Object.hasOwnProperty).replace(
+            'hasOwnProperty',
+            'postMessage'
+          );
+        };
+        window.postMessage = patchedPostMessage;
+      };
+
+      const patchPostMessageJsCode =
+        '(' + String(patchPostMessageFunction) + ')();';
       return (
         <WebView
           {...this.props}
+          injectedJavaScript={patchPostMessageJsCode}
+          javaScriptEnabled={true}
```

Comment:
I believe this is enabled by default. Am I wrong?

Reply:
Yes you're right, let me remove it.